### PR TITLE
EWB-2480: Support protection equipment

### PR DESCRIPTION
**Pull request by Lax125**
Signed-off-by: Marcus Koh <marcus.koh@zepben.com>

# Description

Adds messages and fields to support protection equipment. See spec: https://app.clickup.com/t/6929263/EWB-2473

# Associated tasks:

Tasks blocking this one:
- None

Tasks this is blocking:
- None (for now)

# Checklist:

- [x] I have performed a self review of my own code.
- [x] I have added/updated unit tests for these changes, and if not I have explained why they are not necessary.
- [x] I have updated the changelog.
- [x] I have handled all new warnings generated by the compiler or IDE.
- [x] I have considered if this is a breaking change and have communicated it with other team members if so.

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [EWB-2480 Update evolve-grpc](https://app.clickup.com/t/6929263/EWB-2480)

**Comment by ryanjcoleman** [comment]
Task linked: [EWB-2473 Data model change spec](https://app.clickup.com/t/6929263/EWB-2473)

File: proto/zepben/protobuf/cim/iec61970/base/protection/CurrentRelay.proto [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  /**
+   * The ProtectionEquipment fields for this CurrentRelay.
+   */
+  ProtectionEquipment pe = 1;
+
+  /**
+   * Current limit number one 1 for inverse time pickup in amperes.
+   */
+  double currentLimit1 = 2;
```
Comment (Lax125) [inline proto/zepben/protobuf/cim/iec61970/base/protection/CurrentRelay.proto:30]: There wasn't any unit mentioned in the spec's description. Is this correct?

**Comment by Lax125** [comment]
No tests necessary for a change to the protobuf schema.

**Review by kgreav — APPROVED** [review]
You missed the nc-data.proto changes - i've added them in.

**Pull request merged**